msp/identities.go

```diff
@@ -182,14 +182,16 @@ func (id *identity) Verify(msg []byte, sig []byte) error {
 	}
 
 	if mspIdentityLogger.IsEnabledFor(zapcore.DebugLevel) {
-		mspIdentityLogger.Debugf("Verify: digest = %s", hex.Dump(digest))
-		mspIdentityLogger.Debugf("Verify: sig = %s", hex.Dump(sig))
+		mspIdentityLogger.Debugf("Verify: signer identity (certificate subject=%s issuer=%s serialnumber=%d)", id.cert.Subject, id.cert.Issuer, id.cert.SerialNumber)
+		// mspIdentityLogger.Debugf("Verify: digest = %s", hex.Dump(digest))
```

Comment:
Why drop those?

Reply:
Digest and signature validation are well-trodden. I can't think of anytime in the last five years where this debug has been useful, but it fills the logs with several lines of random bytes. So I thought while adding useful debug I could also remove un-useful debug. I left it commented so that a developer could easily enable if they like. But if you think it is valuable to keep, I can keep.

Reply:
No I trust you if you say that it hasn't been useful - SGTM to remove then